inc/notificationtargetcommonitilobject.class.php

```diff
@@ -55,6 +55,23 @@ function __construct($entity='', $event='', $object=null, $options=array()) {
       $this->options['sendprivate'] = true;
    }
 
+   /**
+    * Get events related to ItiObject
+   **/
+   function getEvents() {
+
+      $events = array('requester_user'    => __('Addition of a user in requesters'),
+                      'requester_group'   => __('Addition of a group in requesters'),
+                      'observer_user'     => __('Addition of a user in observers'),
+                      'observer_group'    => __('Addition of a group in observers'),
+                      'assign_user'       => __('Assignation to a technician'),
```

Comment:
Harmonization of texts:
Addition of a user in technicians
Addition of a group in technicians
Addition of a supplier in technicians